Tests/Marketplace/marketplace_constants.py

```diff
@@ -105,6 +107,13 @@ class GCPConfig(object):
         CORE_PACKS_XPANSE_LIST = packs_list_xpanse.get('core_packs_list')
         CORE_PACKS_XPANSE_LIST_TO_UPDATE = packs_list_xpanse.get('update_core_packs_list')
 
+    with open(os.path.join(os.path.dirname(__file__), VERSIONS_METADATA_FILE), 'r') as server_versions_metadata:
+        VERSIONS_METADATA_CONTENTS = json.load(server_versions_metadata)
+        CORE_PACKS_FILE_VERSIONS = VERSIONS_METADATA_CONTENTS.get('version_map')
```

Comment:
These are not constants.
```suggestion
        versions_metadata_contents = json.load(server_versions_metadata)
        core_packs_file_versions = versions_metadata_contents.get('version_map')
```